app/src/main/java/com/google/ytdl/UploadsListFragment.java

```diff
@@ -46,23 +49,35 @@
  *         Left side fragment showing user's uploaded YouTube videos.
  */
 public class UploadsListFragment extends Fragment implements ConnectionCallbacks,
-        OnConnectionFailedListener {
+        OnConnectionFailedListener, GoogleApiClient.ConnectionCallbacks, GoogleApiClient.OnConnectionFailedListener {
 
     private static final String TAG = UploadsListFragment.class.getName();
     private Callbacks mCallbacks;
     private ImageWorker mImageFetcher;
-    private PlusClient mPlusClient;
+    private GoogleApiClient mGoogleApiClient;
     private GridView mGridView;
+    private static Context mContext;
 
     public UploadsListFragment() {
     }
 
+    @SuppressLint("ValidFragment")
+    public UploadsListFragment(Context context) {
+        mContext = context;
+    }
+
+
     @Override
     public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-        mPlusClient = new PlusClient.Builder(getActivity(), this, this)
-                .setScopes(Auth.SCOPES)
```

Comment:
No problem. I tested it on my phone and the app installed and the mainactivity loads but when it tries and connects, it says Access not configured. The API is not configured for your project or there is a per IP or per-referer restriction configured on your API key and the request does not match those restrictions. Please use the Google Developers Console to update your configuration.

I don't think I have inputted the incorrect API key and i have enabled the APIs needed such as Google+ and Youtube Data 3 - can you let me know if you are experiencing the same issue if you install this app?